README.md

```diff
@@ -162,7 +170,27 @@ The env.example file contains descriptions for all the initial settings. You don
 - [ ] I have transferred the remaining funds of the deployment wallet to the address that originally funded it
   - `make refund`
 
-## Manual deployment (CLI)
+### ERC20 token checklist
+
+When configuring the plugin deployment, make sure to check the implementation of your token contract.
+
+- **Only ERC20s**: The LockManager only deals with underlying tokens of ERC-20 compatible standards. Other fungible token standards such as ERC-1155, are not supported.
+- **Double-entry-point tokens**, i.e. tokens that share the same tracking of balances but have two separate contract addresses from which these balances can be controlled. They should be usable without any issue.
+- **Non-reverting tokens**: ERC-20 Tokens historically handle errors in two possible ways, they either revert on errors or they simply return `false` as a result. SafeERC20 ensures that non reverting tokens do revert given in case of an insufficient balance.
```

Comment:
Consider actively mentioning that the plugin uses SafeERC20, which is why such tokens do not represent a problem to the plugin.

Reply:
It states that the SafeERC20 library does that, which is common knowledge, but it does not state that the plugin uses that library, which requires checking the plugins code.